css/style.css

```diff
@@ -73,14 +73,16 @@ nav.sticky-nav-color {
   padding: 0 0px;
 }
 
-.fixed .nav-inner-most {
-  background: url('/img/lmht-logo.png') no-repeat;
-  display: block;
-  background-position: 10px 0px;
-  background-size: 200px 50px;
-  margin-left: 28px;
-  background-position: left;
+@media all and (min-width: 1140px) {
```

Comment:
Not a blocker but I would assume `all` would mean we do no need `(min-width: 1140px)`.